src/shadows.cpp

```diff
@@ -20,7 +20,6 @@ Shadows::Shadows()
         screen_width = d.width;
```

Comment:
Actually, debug_input when it is instantiated currently fires off a window buffer resize event. Pretty fragile stuff. So I made the window to fire off a resize event after all subsystems are initialized.